.github/workflows/backport-issues.yml

```diff
@@ -0,0 +1,79 @@
+name: Backports
+# This workflow generates "backport" issues when a release branch label is added to an issue
+on:
+  issues:
+    types: [labeled] # triggered when any label is added to an issue
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.event.label.name, 'version/v') }}
```

Comment:
I could be misunderstanding the workflow, but does this handle the case where a version label is being set for the primary issue? If the primary issue is targeting the upcoming release (e.g. `version/v9`), which doesn't need a backport, will this still create a backport issue (effectively a duplicate)?